tensorrt_llm/_torch/attention_backend/sparse/dsa/__init__.py

```
@@ -0,0 +1,31 @@
+    "DSATrtllmAttentionMetadata",
+    "DSAtrtllmAttentionMetadata",
+    "DSACacheManager",
+    "Indexer",
+    "IndexerPrefillChunkMetadata",
+    "RotaryEmbedding",
+    "compute_cu_seqlen_kv_bounds_with_cache",
+    "split_prefill_chunks",
+    "transform_local_topk_and_prepare_pool_view",
+    "_compute_slot_mappings",
+    "rotate_activation",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)